provider/openrouter/openrouter.go

```diff
@@ -81,16 +85,49 @@ type Pricing struct {
 
 // Models returns which models are available to be queried via this provider.
```

Comment:
I guess we never removed the pricing information from the previous approach. Can you please convert the numeric pricing values to floating point right away? https://stackoverflow.com/questions/9452897/how-to-decode-json-with-type-convert-from-string-to-float64